docusaurus.config.js

```diff
@@ -1,106 +1,150 @@
+  baseUrl: "/",
+  onBrokenLinks: "throw",
+  onBrokenMarkdownLinks: "warn",
+  favicon: "img/favicon.ico",
+  organizationName: "zeitgeistpm",
+  projectName: "documentation", // Usually your repo name.
+
+  // support multi-languages
+  i18n: {
+    defaultLocale: "en",
+    locales: ["en", "zh-CN", "ru"],
+    localeConfigs: {
+      en: {
+        htmlLang: "en-GB",
       },
-      items: [
-        {
-          to: '/',  // getting-started',
-          label: 'Basics',
-          position: 'right',
-        },
-        {
-          to: 'prediction-markets',
-          label: 'Learn',
-          position: 'right',
-        },
-        {
-          to: 'guides',
-          label: 'Build',
-          position: 'right',
-        },
-      ],
-    },
-    footer: {
-      style: 'dark',
-      links: [
-        {
-          title: 'Community',
-          items: [
-            {
-              label: 'Website',
-              href: 'https://zeitgeist.pm',
-            },
-            {
-              label: 'Discord',
-              href: 'https://discord.com/invite/xv8HuA4s8v',
-            },
-            {
-              label: 'Twitter',
-              href: 'https://twitter.com/zeitgeistpm',
-            },
-          ],
-        },
-        {
-          title: 'More',
-          items: [
-            {
-              label: 'Blog',
-              to: 'https://blog.zeitgeist.pm',
-            },
-            {
-              label: 'GitHub',
-              href: 'https://github.com/zeitgeistpm',
-            },
-          ],
-        },
-      ],
-      copyright: `Copyright © ${new Date().getFullYear()} Zeitgeist PM`,
-    },
-    prism: {
-      additionalLanguages: ['bash', 'powershell'],
     },
   },
+
+  themeConfig:
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      navbar: {
+        hideOnScroll: true,
+        title: "Zeitgeist Documentation",
+        logo: {
+          src: "img/Zeitgeist-trans.png",
+        },
+        items: [
+          {
+            to: "/", // getting-started',
+            label: "Docs",
+            position: "left",
+          },
+          {
+            type: "docSidebar",
+            sidebarId: "sdk",
+            label: "SDK",
+            position: "left",
+          },
+          {
+            type: "docSidebar",
+            sidebarId: "guide",
+            label: "App Guide",
+            position: "left",
+          },
+          {
+            type: "localeDropdown",
+            position: "right",
+            dropdownItemsAfter: [
+              {
+                href: "https://github.com/facebook/docusaurus/issues/3526",
+                label: "Help Us Translate",
+              },
+            ],
+          },
+          {
+            href: "https://github.com/zeitgeistpm/documentation",
+            position: "right",
+            className: "header-github-link",
+            "aria-label": "GitHub repository",
+          },
```

Comment:
Is this working properly for you? All this does on my machine in create a small whitespace between the localization menu and the search bar that links to our GitHub page. I think this can be deleted.